packages/amplify_authenticator/lib/src/theme/amplify_theme.dart

```diff
@@ -880,6 +880,7 @@ class _AmplifyThemeWrapper {
   Color? get transparent =>
       useAmplifyTheme ? AmplifyColors.transparent : Colors.transparent;
 
-  bool get _isDark =>
-      MediaQuery.platformBrightnessOf(context) == Brightness.dark;
+  bool get _isDark => useAmplifyTheme
```

Comment:
Most of the places `_isDark` only applies when `useAmplifyTheme == true`, however there are a few locations where it is still used when `useAmplifyTheme == false`. That was causing the system brightness to be used, instead of the brightness from MaterialTheme. It resulted in the text on the tabs being the same color as the background if the system brightness and MaterialTheme brightness did not match.

![Screen Shot 2021-11-16 at 4 19 05 PM](https://user-images.githubusercontent.com/20613561/142067815-6d1bbd63-1ad0-4936-8308-1d0fd990bec0.png)